README.md

```diff
@@ -1,2 +1,20 @@
-# hello_world
-Project Hello World week 1
+Project Name
```

Comment:
- [ ] The README looks better but as the previous reviewer mentioned, please consider using the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse, and don't forget to customize it.

Reply:
@Ismailco Indeed I used the readme template, but I deleted some sections. So I understand I shouldn't have deleted any section, but instead just customize the existing sections, rigth?